src/widgets/combo/RequisitesCombo.php

```diff
@@ -34,7 +34,7 @@ public function getPluginOptions($options = [])
                         return data.text;
                     }
 
-                    return '<b>' + data.name + '</b><br>' + data.organization;
+                    return '<b>' + (data.name).replace(/(<([^>]+)>)/ig,'') + '</b><br>' + (data.organization).replace(/(<([^>]+)>)/ig,'');
```

Comment:
```suggestion
                    var name = data.name.replace(/(<([^>]+)>)/ig,'');
                    var organization = data.organization.replace(/(<([^>]+)>)/ig,'');
                    return '<b>' + name + '</b><br>' + organization;
```

уделяй внимание читаемости